users/src/main/java/io/crate/operation/user/UserManagerService.java

```diff
@@ -63,6 +63,13 @@
         }
     };
 
+    private static final Consumer<User> ENSURE_PRIVILEGE_USER_NOT_SUPERUSER = user -> {
```

Comment:
I think the name is misleading. `VALIDATE_USER_IS_NOT_SUPERUSER` might make more sense? (as only the exception message is about privileges)

Also, maybe just a personal preference, but do we really need a consumer? (an instance or even static method will do)